### DISCUSSION TITLE
Make the turtle an actual turtle

### DISCUSSION THREAD

**Discussion by Dinock** [GDQuest/learn-gdscript#142] [Ideas]
I would suggest to make the turtle an actual (adorable) turtle and not just a plain symbol.
Displaying a (simple but cutely drawn) turtle could help make students more emotionally "attached" or involved, which could lead to being more motivated to make the turtle actually move (and thus learn how to code).

Calling something a turtle which is obviously just a symbol, was, at least for me, quite irritating, especially when the turtle was introduced in lesson 4.

Or is the turtle some kind of inside joke, similar to ["Hello, World!"](https://en.wikipedia.org/wiki/%22Hello,_World!%22_program)?

**Comment by NathanLovato** [comment]
Sure, the current icon is a placeholder
